### ISSUE TITLE
Non-visible tiles with light on them look like space tiles

### ISSUE THREAD

**Issue by Shroopy** [tgstation/tgstation#62837]
## Round ID:
[173080](https://scrubby.melonmesa.com/round/173080)

## Testmerges:
#62727, #62711

## Reproduction:
Wear night vision goggles or have someone with a light stand next to a tile that you can't see (for example, is blocked by a wall for you but not the light), and you'll see the space background instead of black.

![image](https://user-images.githubusercontent.com/46693163/141734791-ef4a422e-840a-414d-9054-b7e1214d39d7.png)
![image](https://user-images.githubusercontent.com/46693163/141734832-051dd0b9-b0e5-405e-aeec-38cc86be2fab.png)

**Comment by LemonInTheDark** [comment]
Dupe of https://github.com/tgstation/tgstation/issues/61756, likely a won't fix because it looks pretty, and because fixing it requires digging around with byond funtimes that none knows enough about
